package.json

```diff
@@ -40,6 +42,7 @@
     "typestrict": "^0.0.8"
   },
   "dependencies": {
+    "express": "^4.16.3",
```

Comment:
mmh we cannot import `express` as a dependency here (this library is also used by the app) - is there a way to just import the type defs and not the library itself? it looks like we just need the `Response` type.

Reply:
I've removed express from deps